capstone/src/main/python/main.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import pandas as pd
+from sklearn.tree import DecisionTreeRegressor
+from google.cloud import datastore
+from flask import Flask
+app = Flask(__name__)
+
+
+@app.route('/regression-analysis')
+def main():
+    df = load_dataframe()
+    fit_regressor = train_model(df)
+    add_predictions(fit_regressor)
+
+
+def load_dataframe():
+    datastore_client = datastore.Client()
+    query = datastore_client.query(kind='Response')
+    results = list(query.fetch())
+    size = len(results)
+
+    # Create Data Frame from entities
+    directExperiences = []  # Experience with police (yes or no)
```

Comment:
Consider keeping these variables consistent and use underscores. For example `datastore_client` uses underscores but `directExperiences` uses camel case.

Reply:
I'll add these changes